zubhub_frontend/zubhub/package.json

```diff
@@ -47,7 +47,7 @@
   },
   "scripts": {
     "analyze": "source-map-explorer 'build/static/js/*.js'",
-    "start": "react-scripts start",
+    "start": "react-scripts --openssl-legacy-provider start",
```

Comment:
Why do we need this?

Reply:
using `"start": "react-scripts start"` generated the following error for me:
> Error: error:0308010C:digital envelope routines::unsupported

I looked for solutions [here](https://stackoverflow.com/questions/69692842/error-message-error0308010cdigital-envelope-routinesunsupported) and found that the error can be resolved by using `"start": "react-scripts --openssl-legacy-provider start"`

This change, however, should have been made locally only and should not have been committed.
I will reset the change by making another commit.